### DISCUSSION TITLE
Auto collapse left sidebar (2.5)

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#8120] [Ideas]
> Created by: Nathan Doyle (natetronn@gmail.com) on 2015/03/11 04:20:14 +0000
> Votes at time of UserVoice import: 30

-----------------------------------

This FR was requested in Slack.

"Any thought on collapsing the sidebar by default? I feel this results in much less cognitive load when dealing with several columns and long lists!" -- jwb009

https://craftcms.slack.com/files/jwb009/F0E18PBHV/customized-entry-index_edit.png

Example: http://www.gv.com

**Comment by angrybrad** [comment]
> Posted by Johannes Lamers (johanneslamers@gmail.com) on 2015/03/10 17:03:25 +0000
-----------------------------------

Made a little cp.css hack. So it&#39;s working already. 

https://github.com/johanneslamers/craft-collapse

**Comment by angrybrad** [comment]
> Posted by Matt Jenkins (matt@artdepartment.co.uk) on 2016/01/29 09:26:36 +0000
-----------------------------------

Having this as a variable for the global sidebar would be fantastic, even if it collapsed to the burger menu that would be fine.

**Comment by angrybrad** [comment]
> Posted by David Ritter (david.ritter@spinthis.com) on 2015/05/18 19:35:25 +0000
-----------------------------------

Agree wholeheartedly with Mr. DiLoreto; that Pandadoc is exactly what we need. 

Especially when editing entries, it&#39;d be nice to gain as much space as possible. 2.4 got out of your way in that respect. That extra 220px becomes especially valuable when seeing the titles/content of  collapsed matrix blocks.

**Comment by angrybrad** [comment]
> Posted by Brett Burwell (bburwell@thisisstatic.com) on 2015/03/18 21:23:48 +0000
-----------------------------------

I agree with Patrick and Stuart. 

The video clip Lindsey included is a nice working example of how that might look / function. Thanks for sharing!

**Comment by angrybrad** [comment]
> Posted by Stuart McCoy (stuart@meltingcrayons.com) on 2015/03/11 01:51:01 +0000
-----------------------------------

I agree with Patrick. The default should be uncollapsed but there should be the ability to collapse the menu. Dare I say, I&#39;d like to see it collapse to a single column of icons a la WordPress. This allows users to quickly switch from one panel to another without having to first uncollapse the menu. The site&#39;s favicon could be used for the site link at the top of the menu.

**Comment by angrybrad** [comment]
> Posted by Anonymous (allan@vectorliving.com) on 2015/12/07 23:31:24 +0000
-----------------------------------

I&#39;d also, be happy with moving it to the right side.

**Comment by angrybrad** [comment]
> Posted by Bart Rylant (bart.rylant@gmail.com) on 2015/10/09 13:54:51 +0000
-----------------------------------

Please please please. I was at a potential client today showing Craft on an iPad, and even in landscape mode it was showing all entries in a mobile layout (with everything stacked instead of a table). Didn&#39;t make a good impression.

**Comment by angrybrad** [comment]
> Posted by Chris Mousdale (chris@mousdale.com) on 2015/12/02 09:02:08 +0000
-----------------------------------

Agreed. I would actually like the option to move the sidebar back to the top!

**Comment by angrybrad** [comment]
> Posted by Lindsey DiLoreto (lindsey@doublesecretagency.com) on 2015/03/11 04:48:18 +0000
-----------------------------------

PandaDoc does this, and it&#39;s wonderful... http://screencast.com/t/d4HE3KMW1RUF

**Comment by angrybrad** [comment]
> Posted by Lindsey DiLoreto (lindsey@doublesecretagency.com) on 2015/03/11 04:48:18 +0000
-----------------------------------

Brilliant idea, I absolutely love this!

RE: When to collapse

My instinct says it should be expandable/collapsible at will. Exactly like Matrix Blocks, the system would remember &quot;how you had it last&quot;, and wouldn&#39;t affect other users.

**Comment by mark-chief** [comment]
Fantastic Idea and thanks for the cp.css! Would love to see this rolled out.
